### PR TITLE
Added new manager page

### PR DESCRIPTION
**Pull request by calvinbrown085**
Updated manager page

### CODE REVIEW

File: templates/Managerpage.html [hunk 1]
```diff
@@ -12,18 +12,28 @@
 <form action="/managerSearch" method="post">
 <center><b>Generate Product orders</b></center>
 <p>
```
Comment (ethanphunter) [inline templates/Managerpage.html:14]: What would happen if you took this `<p>` tag out? or at least close it? Would that make the button happy again?

Comment (calvinbrown085) [inline templates/Managerpage.html:14]: @ethanphunter it did not :(

Comment (ethanphunter) [inline templates/Managerpage.html:14]: Dang it

Comment (ethanphunter) [inline templates/Managerpage.html:14]: What if we shrink the form down to only have the `<input>` tag in it?

Comment (ethanphunter) [inline templates/Managerpage.html:14]: Oh wait I know

File: templates/Managerpage.html [hunk 2]
```diff
@@ -12,18 +12,28 @@
 <form action="/managerSearch" method="post">
 <center><b>Generate Product orders</b></center>
```
Comment (ethanphunter) [inline templates/Managerpage.html:13]: This probably won't affect anything, but the `<b>` tag is only supposed to be used as a last resort now in html 5. They suggest to use css instead. Probably won't fix the button though

**Pull request merged**